examples/flow_benchmark_2d_case_1.py

```diff
@@ -0,0 +1,114 @@
+"""
+This module contains the implementation of Case 1 from the 2D flow benchmark [1].
+
+Note:
+    We provide the two variants of this benchmark, i.e., Case 1a (conductiv fractures)
+    and Case 1b (blocking fractures). The former is provided by the class
+    `FlowBenchmark2dCase1aModel` whereas the latter by the class
+    `FlowBenchmark2dCase1bModel`.
+
+References:
+    - [1] Flemisch, B., Berre, I., Boon, W., Fumagalli, A., Schwenck, N., Scotti, A.,
+      ... & Tatomir, A. (2018). Benchmarks for single-phase flow in fractured porous
+      media. Advances in Water Resources, 111, 239-258.
+
+"""
+import numpy as np
+
+import porepy as pp
+from porepy.applications.discretizations.flux_discretization import FluxDiscretization
+from porepy.models.constitutive_laws import DimensionDependentPermeability
+
+solid_constants = pp.SolidConstants({"residual_aperture": 1e-4})
```

Comment:
I believe we're missing the `normal_permeability` declaration in the solid constants.

Reply:
You're right. And we have the related question of how to define tangential fracture permeability. This prompted me to think a bit on the risk involved in the most tempting solution, which would be passing them in self.params. To avoid the user misspelling the parameter name, forgetting to specify it or otherwise get confused, I introduced tailored SolidConstants. Please let me know if you approve of the approach.

Reply:
I support 100% this approach. Especially, when it comes to benchmarks, where the number 1 priority is to reproduce the setup.

Great job!